packages/siaviewnode-client/src/components/Dashboard.tsx

```diff
@@ -0,0 +1,161 @@
+/** @jsx jsx */
+import { jsx } from "theme-ui"
+import { Container } from "@material-ui/core";
+
+// React Grid System
+import { Row, Col } from 'react-grid-system';
+
+// D3 / Recharts charts library
+import { BarChart, Bar, XAxis, YAxis, CartesianGrid, Tooltip, Legend, Text, ResponsiveContainer} from 'recharts';
+
+// Font Awesome
+import { faThumbtack, faUpload, faDownload, faServer, faStopwatch, faMoneyBillAlt } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { theme } from "../../theme/theme";
+import { icon } from "@fortawesome/fontawesome-svg-core";
+
+
+export const Dashboard = (props) => {
+
+    const stats = props.api
+
+    return (<div style={{ width: "100%", paddingTop: 50}}>
+        <Container>
+            <Row justify="between">
+                <StatBox title="Pinned files" icon={faThumbtack} iconTransform="rotate-45" stat={stats.total_files_pinned} suffix=""/>
+                <StatBox title="Stored data" icon={faServer} iconTransform="" stat="0.145" suffix="TB"/>
+                <StatBox title="Files served this week" icon={faUpload} iconTransform="" stat={stats.total_files_served[168]} suffix=""/>
```

Comment:
The 168 number is slightly magical. I get that it's hours and it's OK to use it like that but it might be a little easier on the reader if we moved all those (1, 24, 168, 720, 2160) to a set of constants with descriptive names. Just to remove a little cognitive load.